### DISCUSSION TITLE
Not able to change FROM EMAIL field

### DISCUSSION THREAD

**Discussion by vikram-chauhan-7** [emailjs-com/help#114] [Q&A]
Hi,
When I changed FROM EMAIL field from default one to some other email id. We started getting below issue while sending the mail.  I want mail should be deliver from a particular mail id not from default one.

<img width="748" alt="image" src="https://github.com/emailjs-com/help/assets/137787286/633b7f22-a646-4fed-ae96-32c4aeca4d00">

**Comment by xr0master** [comment]
Probably duplicate https://github.com/orgs/emailjs-com/discussions/8
